op_builder/builder.py

```diff
@@ -267,6 +273,12 @@ def cpu_arch(self):
             return '-mcpu=native'
         return '-march=native'
 
+    def is_cuda_available(self):
+        if torch.cuda.is_available():
```

Comment:
Will this not throw an import exception if torch.cuda is not available? Should this be in a try/except?

Reply:
good point 👍 I will add that. Thanks